### PR TITLE
Update ResolvedTitle.md

### PR DESCRIPTION
**Pull request by glpayson**
**Old**: https://github.com/careerbuilder/DataScienceAPIDocumentation/blob/master/ResolvedTitle.md
**New**: https://github.com/careerbuilder/DataScienceAPIDocumentation/blob/rts_translate/ResolvedTitle.md

### CODE REVIEW

File: ResolvedTitle.md [hunk 1]
```diff
@@ -1,46 +1,98 @@
...
+    </tr>
+</table>
+
+### Version Parameter:
+
+This API is in transition to match CareerBuilder API standards.  To provide peaceful transition, we support versions 0.9 and 1.0.  Version is passed in the Accept header as the parameter version (Ex. `Accept:application/json;version=1.0`). 
+
+If the version is not explicitly specified in the Accept header then the service defaults to version 0.9.  We **highly encourage** new users to pass version 1.0 in the Accept header. 
+
+In version 0.9, all input parameters and output json fields are in camel case.  In version 1.0, input parameters and output json fields are in snake case (lowercase with underscores).  Version 0.9 is scheduled for retirement by the end of Q1 2017.
```
Comment (gliu6) [inline ResolvedTitle.md]: Q1 has passed long, maybe saying Q2?

**Review by gliu6 — APPROVED** [review]
lovely

**Comment by gliu6** [comment]
👍 still lovely

File: ResolvedTitle.md [hunk 2]
```diff
@@ -1,46 +1,81 @@
...
+    </tr>
+    <tr>
+        <td><b>Name</b></td>
+        <td><b>Required</b></td>
+        <td><b>Comment</b></td>
+    </tr>
+    <tr>
+        <td><pre>target_titles</pre></td>
+        <td>true</td>
+        <td>A pipe delimeted list of possible titles to resolve to.  A single title may be supplied if the user simply wishes to test whether an input is a good match for a specified title.</td>
```
Comment (thomasmahoney) [inline ResolvedTitle.md]: typo here ("delimeted")

File: ResolvedTitle.md [hunk 3]
```diff
@@ -1,46 +1,81 @@
...
+        <tr>
+        <td><pre>raw_description</pre></td>
+        <td>false</td>
+        <td>The description of the job offer, possibly empty.</td>
+    </tr>
+</table>
+
+### Version Parameter:
+
+*Resolved Title Service* is currently on version 1.0.  Version **MUST BE** specified the Accept header (Ex. `Accept:application/json;version=1.0`).
```
Comment (thomasmahoney) [inline ResolvedTitle.md]: another typo ("specified the")

**Review by thomasmahoney — APPROVED** [review]

**Pull request merged**